packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/new-account-form-controller.js

```diff
@@ -300,17 +294,29 @@ export default class NewAccountFormController {
     let promise = this.userAccountServiceInfos
       .updateUseraccountInfos(model)
       .then((result) => {
+        const tracking = {
+          name: `dedicated::account::user::infos_${
+            result !== 'null' ? 'error' : 'success'
+          }`,
+          type: 'navigation',
+        };
+        if (!this.coreConfig.isRegion('US')) {
```

Comment:
you add feature flipping for `creation`. Why don't use it here ?

Reply:
I was consistent with the email consent part (line 155) but indeed i can use feature-flipping hete